src/components/AppRoutes.js

```diff
@@ -25,39 +39,41 @@ const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
 
 const AppRoutes = () => (
   <SentryRoutes>
-    <Route path="/auth/callback" element={<Callback />} />
+    <Suspense fallback={<></>}>
```

Comment:
This wasn't working when I tested locally - but I think if you wrap the individual elements inside each `Route` with the `Suspense` as per [this article](https://www.robinwieruch.de/react-router-lazy-loading/) instead of all the routes, it should fix it